### ISSUE TITLE
Could you please offer a full sort version of DisenCDR?

### ISSUE THREAD

**Issue by ShadowTinker** [cjx96/DisenCDR#4]
Hi!

Thanks for your great work. I'm really interested in DisenCDR. I notice that the model is evaluated by sampling 999 neg samples, which may inherit from BiTGCF. But to my best knowledge full sort evaluation is a more common and reasonable protocol and it's not very time consuming by utilizing an inner-product predictor, which is exactly used by DisenCDR. I've tried to implement this. But the results on the training set are poor, not to mention the results on the test set.

So I wonder whether you could offer a full sort version of DisenCDR?

Thanks a lot!
